src/Model/FormationManager.php

```diff
@@ -23,4 +23,12 @@ public function selectAllByDiscipline(int $id): array
 
         return $statement = $statement->fetchAll();
     }
+
+    //fonction pour supprimer une formation
+    public function delete(int $id): void
```

Comment:
you already have a delete method from your abstractManager parent class, use it to avoid code duplication

Reply:
yeah right exactly

Reply:
ok, fixed